com.kristianduske.TrenchBroom.json

```diff
@@ -0,0 +1,144 @@
+                {
+                    "type": "archive",
+                    "url": "http://downloads.sourceforge.net/freeimage/FreeImage3180.zip",
+                    "sha256": "f41379682f9ada94ea7b34fe86bf9ee00935a3147be41b6569c9605a53e438fd"
+                },
+                {
+                    "type": "patch",
+                    "path": "freeimage-fix-install.patch",
+                    "strip-components": 0
+                }
+            ]
+        },
+        {
+            "name": "pandoc",
+            "only-arches": [
+                "x86_64"
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp bin/pandoc /app/bin/pandoc",
+                "cp bin/pandoc-citeproc /app/bin/pandoc-citeproc"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/jgm/pandoc/releases/download/2.2/pandoc-2.2-linux.tar.gz",
+                    "sha256": "06ecd882e42ef9b7390b1c82e1e71b3ea48679181289b9b810a8797825bed8ed"
+                }
+            ]
+        },
+        {
+            "name": "trenchbroom",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_CXX_FLAGS=-Werror",
+                "-DwxWidgets_PREFIX=/app",
+                "-DOpenGL_GL_PREFERENCE=GLVND"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/kduske/TrenchBroom.git",
+                    "tag": "v2019.2",
+                    "commit": "67d4016c0b8e8a6cc6cbb11eb6b9ddf2779548ed"
+                },
+                {
+                    "type": "script",
+                    "commands": [
+                        "TB_DEV_MODE=1 trenchbroom $@"
+                    ],
+                    "dest-filename": "trenchbroom.sh"
+                },
+                {
+                    "type": "file",
+                    "path": "com.kristianduske.TrenchBroom.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/kduske/TrenchBroom/v2019.2/app/resources/linux/trenchbroom.desktop",
+                    "sha256": "d9f533982b9d3d6be4b33e1a3248c7223522654da4f17c6190a408cb2e6dd2b6"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/kduske/TrenchBroom/raw/master/app/resources/linux/icons/icon_256.png",
+                    "sha256": "9edb0d713fde4c7417fa4b3f7037974a1dedb9e3099e1202a8b744ed6f30b314"
+                }
+            ],
+            "post-install": [
+                "install -D trenchbroom.sh /app/bin/trenchbroom.sh",
+                "mkdir -p /app/share/applications",
+                "install -D trenchbroom.desktop /app/share/applications/",
```

Comment:
Better try this way:
```
install -Dpm0644 trenchbroom.desktop /app/share/applications/${FLATPAK_ID}.desktop
```

Reply:
Ok, I changed it like that after looking from `man` what those options do.